### PR TITLE
More minor fixes to pave the way for fixing @LinkingObjects UTs

### PR DESCRIPTION
**Pull request by bmeike**
A couple more trivial cleanups that can go into releases, to pave the way fixing the @LinkingObjects UT.

### CODE REVIEW

File: realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java [hunk 1]
```diff
@@ -547,10 +561,10 @@ public void migration_backlinkedFieldInUse() {
      *     private final RealmResults<XXXBacklinksSource> parents = null;
      * }
      * </pre>
-     * If the both classes were used in the configuration, the test would fail because of the
-     * missing class.  Since the configuration contains only the single class `BacklinksTarget`,
-     * basic validation passes.  Backlink validation, however, should fail, seeking the
-     * `BacklinksSource` table.
+     * If the both classes were present in the configuration, the test would fail because there
```
Comment (cmelchior) [inline realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java]: `if the both` -> `if both` ?

Comment (bmeike) [inline realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java]: Whoops.  Yep!

File: realm/realm-annotations-processor/src/test/java/io/realm/processor/RealmBacklinkProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2014-2017 Realm Inc.
```
Comment (beeender) [inline realm/realm-annotations-processor/src/test/java/io/realm/processor/RealmBacklinkProcessorTest.java]: new file should use the year which created the file i think `Copyright 2017 Realm Inc.`

Comment (bmeike) [inline realm/realm-annotations-processor/src/test/java/io/realm/processor/RealmBacklinkProcessorTest.java]: yep.  will fix.

**Pull request merged**